lib/icedfrisby.js

```diff
@@ -778,6 +724,24 @@ class Frisby {
   }
 
 
+  /**
+   * @param {object} options object
+   *
+   * @desc Set configuration options on this object.
+   * inspectOnFailure: This is a really neat option that will help you figure out what is happening with your requests. Dumps request/response information to the logs.
+   * json: Sets body to JSON representation of value and adds Content-type: application/json header. Additionally, parses the response body as JSON.
+  */
+  config (opts) {
+    if ('inspectOnFailure' in opts) {
+      this._inspectOnFailure = opts.inspectOnFailure
+    }
+    if ('json' in opts) {
+      this.current.request.json = opts.json
```

Comment:
Should this just be checking opts is an object and merging it into current.request?

Reply:
No. No it shouldn't.